index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const cors = require('cors');
+const { pool } = require('./config');
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(cors());
```

Comment:
Cheers to using CORS!  Just as a small note, the way it's setup, anyone anywhere can make network requests to your BE.  You may want to add some additional configuration at some point.